conda/plugins/types.py

```diff
@@ -141,3 +141,12 @@ class CondaAuthHandler(NamedTuple):
 
     name: str
     handler: type[ChannelAuthBase]
+
+
+class CondaHealthChecks(NamedTuple):
```

Comment:
The class name should be singular since each health check plugin would be have exactly one instance of this class.

```suggestion
class CondaHealthCheck(NamedTuple):
```